src/sentry/issues/occurrence_consumer.py

```diff
@@ -125,6 +125,12 @@ def _get_kwargs(payload: Mapping[str, Any]) -> Mapping[str, Any]:
             if payload.get("culprit"):
                 occurrence_data["culprit"] = payload["culprit"]
 
+            if payload.get("initial_issue_priority"):
```

Comment:
nit: in theory `0` is a valid priority, which is falsy.


```suggestion
            if payload.get("initial_issue_priority") is None:
```